docs/solar/README.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Solar API Overview
+---
+
+# Solar API Endpoints
+
+The base URI for all site-specific solar APIs is `https://owner-api.teslamotors.com/api/1/energy_sites/{site_id}/` followed by the endpoint name and parameters. `site_id` identifies the solar installation.  See the [products.md](products.md) file for obtaining the `site_id` value.
```

Comment:
```suggestion
The base URI for all site-specific solar APIs is `https://owner-api.teslamotors.com/api/1/energy_sites/{site_id}/` followed by the endpoint name and parameters. `site_id` identifies the solar installation.  See the [`products` endpoint](products.md) for obtaining the `site_id` value.
```

You should link to logical titles/names of documents, not their files. Most users aren't browsing the repository, they're reading the Gitbook site.